### DISCUSSION TITLE
Polyhedra mesh problem

### DISCUSSION THREAD

**Discussion by EduLarrubia** [mdolab/dafoam#497] [General]
I'm attempting to use the Ansys-Fluent polyhedra mesh in DAFoam, but I haven't been successful so far. I'd like to know if this combination is incompatible or if I'm missing something. Has anyone else experienced the same issue? Thank you.

![MicrosoftTeams-image (1)](https://github.com/mdolab/dafoam/assets/147490279/b077cfcd-080a-47af-b549-801d32520856)

**Comment by friedenhe** [comment]
Have you tried this? https://www.cfd-online.com/Forums/main/246961-ansys-fluent-polyhedral-mesh-openfoam.html#post842319

**Reply by EduLarrubia** [reply]
Thank you for the answer. I didn't have a problem importing the mesh and converting it to OpenFOAM. The simulations run smoothly. The issue only arises when I use DAFoam. Do you know if this type of mesh is incompatible with the DAFoam algorithm?

**Comment by friedenhe** [comment]
DAFoam supports the OpenFOAM mesh type. So as long as you can run the native OpenFOAM solver, it should work for DAFoam. Check the last two sentences of this FAQ and see if that fixes your issue. https://dafoam.github.io/mydoc_get_started_faq.html#can-i-use-my-own-mesh

Basically, your DAFoam log says the mesh quality check failed (it returned nan), so it is likely that your FFD did not fully cover your design surface.
